eth/gaspricemonitor_test.go

```diff
@@ -131,14 +132,21 @@ func TestStart_Polling(t *testing.T) {
 	// There should be more queries now
 	assert.Greater(gpo.Queries(), queries)
 	assert.Equal(gasPrice3, gpm.GasPrice())
+
+	go func() {
+		gpo.SetGasPrice(gasPrice4)
+	}()
+	time.Sleep(100 * time.Millisecond)
+	assert.Greater(gpo.Queries(), queries)
```

Comment:
I think we want to update `queries` with `queries = gpo.Queries()` before L136 in order to take a snapshot of the # of queries before and after the sleep to ensure that an additional query was ran during the sleep and after the goroutine executed.